dashboard/lib/tasks/seed_in_test.rake

```diff
@@ -1,6 +1,13 @@
 Rake::Task['db:test:prepare'].enhance do
   ActiveRecord::Base.establish_connection(:test)
   Rake::Task['db:fixtures:load'].invoke
+  require 'cdo/db_utils'
+  DBUtils.reload_proxy_backends
   Rake::Task['seed:test'].invoke
   ActiveRecord::Base.establish_connection(ENV['RAILS_ENV'].to_sym)
 end
+
+Rake::Task['db:test:purge'].enhance do
+  require 'cdo/db_utils'
+  DBUtils.reload_proxy_backends
+end
```

Comment:
These two operations caused issues with the proxysql connections:
- The `DROP DATABASE` / `CREATE DATABASE` operations from `db:test:purge` interfered with the db-connection schema caching proxysql managed, causing `'No database selected'` to be returned (a known issue).
- The `SET FOREIGN_KEY_CHECKS = 0` session variable change from `db:fixtures:load` caused something else to break in a way I couldn't figure out how to fix.

In both of these cases the easiest workaround I was able to find was to restart the proxy service. Since these particular operations are only performed during CI builds this workaround should be safe enough for now (and we can track/follow-up on these issues in the future).